Makefile

```diff
@@ -188,6 +185,11 @@ $(MODULES)/*.inc: $(TOOLS)/embed $(MODULES)/*.wren
 
 modules: $(MODULES)/*.inc
 
+$(OBJS)/libwren.o:
+	@mkdir -p $(OBJS)
+	@echo "==== Building wren module ===="
+	$(CC) -c $(INCLUDES)/wren.c -o $(OBJS)/libwren.o $(IFLAGS)
```

Comment:
Do we need to pass the WREN_PARAMS in here, to ensure we get the correct optional modules?

Reply:
Fixed in 4847c6c